x/globalfee/ante/fee.go

```diff
@@ -186,11 +215,15 @@ func (mfd FeeDecorator) DefaultZeroGlobalFee(ctx sdk.Context) ([]sdk.DecCoin, er
 }
 
 func (mfd FeeDecorator) getBondDenom(ctx sdk.Context) string {
-	var bondDenom string
-	if mfd.StakingSubspace.Has(ctx, stakingtypes.KeyBondDenom) {
-		mfd.StakingSubspace.Get(ctx, stakingtypes.KeyBondDenom, &bondDenom)
+	// prevent the getter above to panic
+	// when the staking subspace isn't set
+	if !mfd.StakingSubspace.HasKeyTable() {
```

Comment:
HasKeyTable just registers the params and the way to valid the params ? we do not need this ?